x-pack/plugins/ml/public/application/components/data_grid/data_grid.scss

```diff
@@ -3,3 +3,9 @@
     text-transform: none;
   }
 }
+
+// Make sure the charts tooltip in popover
+// have higher zIndex than Eui popover cells
+[id^='echTooltipPortal'] {
+  z-index: 10000;
```

Comment:
This works for me only when adding `!important` as otherwise it gets overridden by the inline style.

Reply:
Thanks Pete, this is now updated here https://github.com/elastic/kibana/pull/91061/commits/3f76bf89d52e57b957b1f56ebf080d675815760f